### PR TITLE
Add basic System.Resources.ResourceManager tests

### PR DESCRIPTION
**Pull request by mellinoe**
This adds some very basic coverage of the types in the library.

Note: I had some tests that grabbed resources from a different culture, exercising ResourceManager.GetString(string, CultureInfo), but it seems like our test script isn't copying over all of the satellite assemblies right now, so I had to manually copy them into my test directory. I think that will require some buildtools changes to get working, so I left them out for now.

### CODE REVIEW

**Comment by ellismg** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
@dotnet-bot test this please

File: src/System.Resources.ResourceManager/tests/MissingManifestResourceExceptionTests.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using Xunit;
```
Comment (stephentoub) [inline src/System.Resources.ResourceManager/tests/MissingManifestResourceExceptionTests.cs]: These files are all missing the license header.

**Comment by stephentoub** [comment]
Other than the lack of the right headers, LGTM.

**Comment by mellinoe** [comment]
Oops, completely slipped my mind... Fixing...

**Comment by mellinoe** [comment]
Should be good now.

**Comment by Maxwe11** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
LGTM

**Pull request merged**